include/networkit/graph/GraphTools.hpp

```diff
@@ -129,13 +129,36 @@ std::pair<node, node> size(const Graph &G) noexcept;
 double density(const Graph &G) noexcept;
 
 /**
- * Returns the volume (sum of the out-degree of all nodes) of the graph.
+ * Returns the volume of the input graph.
  *
  * @param G The input graph.
  * @return Volume of the graph.
+ * 
  */
 double volume(const Graph &G);
 
+/**
+ * Returns the volume (sum of the in-degree of all nodes) of a subgraph 
+ * defined by a set of nodes. If the set is empty, the volume of the complete
+ * graph is computed.
+ *
+ * @param G The input graph.
+ * @param nodes Subset of nodes.
+ * @return inVolume of the subgraph.
+ */
+double subgraphInVolume(const Graph &G, const std::unordered_set<node> &nodes);
```

Comment:
For the input nodes, can we use ranges? So the user is not forced to create an `unordered_set`.